source/content/integrated-composer.md

```diff
@@ -19,6 +19,21 @@ Please note the Limited Availability program does not include a path to upgrade
 
 <Partial file="drupal-9-upstream-install.md" />
 
+### Drupal 8 with Integrated Composer
+
+1. Follow step 1 of the [Drupal 9](#drupal-9-with-integrated-composer) section above.
+
+1. Keep the site's Development Mode set to Git, and [clone the site locally](/local-development#get-the-code).
+
+1. From a terminal working in the "upstream-configuration" directory, update Composer's version requirements for the project:
+
+  ```bash{outputLines: 2}
+  composer require drupal/core-composer-scaffold:^8.8 \
```

Comment:
We don't need to tell folks to back up to the older version of core-composer-scaffold; they're better off staying on ^9 for this one.